package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "altis-consent-api",
+  "version": "1.0.2",
+  "description": "Javascript Consent API to read and register the current consent category.",
+  "main": "plugin.php",
```

Comment:
This doesn’t really make sense for a package.json file to reference a php file. Perhaps it’s ok to just remove this